### ISSUE TITLE
Devportal Try out improvement 

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#6863]
If a user selects JWT, they should still be able to try out the API without manually copying the token, the token should be stored as a masked text so that user cannot copy the token from the tryout field

Issues raised by Usability Testing candidates
